addons/website/models/website.py

```diff
@@ -1773,6 +1773,10 @@ def _basic_enumerate_words(self, search_details, search, limit):
             domain.append(OR(fields_domain))
             domain = AND(domain)
             records = model.search_read(domain, fields, limit=1000)
+            if len(records) == 1000:
```

Comment:
Could we make those 2 lines `1000` "linked" by using a variable? Will be easier to figure.
And definitely add a comment for this new code block, write it as you prefer but something along those lines I guess
```suggestion
            perf_limit = 1000
            records = model.search_read(domain, fields, limit=perf_limt)
            # Exact match might have been missed due to the 1000 blabla.. (I let you write it the way you prefer)
            if len(records) == perf_limit:
```